Formula/sipsak.rb

```diff
@@ -5,6 +5,11 @@ class Sipsak < Formula
   sha256 "c6faa022cd8c002165875d4aac83b7a2b59194f0491802924117fc6ac980c778"
   license "GPL-2.0-or-later"
 
+  livecheck do
+    url :homepage
+    strategy :github_latest
+  end
+
```

Comment:
Why do you need this?

Reply:
Sorry, I've mixed something up.
I'll revert it.